src/hooks/useMembership.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+
+export const MEMBERSHIP_URL = process.env.MEMBERSHIP_URL;
+
+type Membership = {
```

Comment:
I need to update the contract here in order to being able to integrate properly.